Subject: Remove the length check
From: Juergen Beisert <jbe@pengutronix.de>

Remove the length check. It prevents handling of long paths. Program ends with
a confusing error message like this one:

"serious libopt error prevented parsing library names.  Invalid input to
libopt is NOT the problem."

The message is correct, but does not help...

Signed-off-by: Juergen Beisert <jbe@pengutronix.de>

---
# wsa: 20091230: fixed upstream: http://netpbm.svn.sourceforge.net/viewvc/netpbm?view=rev&revision=118

 buildtools/libopt.c |    4 ----
 1 file changed, 4 deletions(-)

Index: buildtools/libopt.c
===================================================================
--- buildtools/libopt.c.orig
+++ buildtools/libopt.c
@@ -509,9 +509,6 @@ main(int argc, char **argv) {
             runtime = TRUE;
         else if (strcmp(argv[arg], "-quiet") == 0)
             quiet = TRUE;
-        else {
-            if (strlen(argv[arg]) > 200)
-                error = TRUE;
             else {
                 const char * options;
                 processOneLibrary(argv[arg], runtime, explicit, 
@@ -527,7 +524,6 @@ main(int argc, char **argv) {
                     strfree(options);
                 }
             }
-        }
     }
     if (error) {
         fprintf(stderr, "serious libopt error prevented parsing library "
